flink-python/src/main/resources/META-INF/NOTICE

```diff
@@ -59,7 +59,6 @@ See bundled license files for details
 - com.google.protobuf:protobuf-java:3.7.1
 - com.google.protobuf:protobuf-java-util:3.7.1
 - com.google.auth:google-auth-library-credentials:0.13.0
-- cloudpickle:1.2.2
```

Comment:
this entry needs to be moved to `tools/releasing/NOTICE-binary_PREAMBLE.txt`, otherwise this will not show up in any `NOTICE`.

Reply:
I assume the py4j entry is retained since we still bundle the java part?

Reply:
Yes. The `flink-python` jar bundles the java part of `py4j`.